R/filter_log_output_types.R

```diff
@@ -0,0 +1,20 @@
+filter_log_output_types <- function(log_list, output_file_types = c()) {
```

Comment:
this... could be better - i spent some time on it and it was the best i could come up with. it's not terrible, and it works... 🤷‍♀